### PR TITLE
crew: Add `sysinfo` command

### PR DESCRIPTION
**Pull request by supechicken**
Add a command for generating system info for the issue template

Tested on `x86_64`

### CODE REVIEW

File: bin/crew [hunk 1]
```diff
@@ -1798,6 +1806,31 @@ def search_command(args)
...
+  if File.exist?('/etc/lsb-release')
+    lsb_release = File.read('/etc/lsb-release').scan(/^(.+?)=(.+)$/).to_h
+  else
+    # newer version of Chrome OS exports info to env by default
+    lsb_release = ENV
+  end
+
+  puts <<~MD
+    - Architecture: `#{ARCH_ACTUAL}` (`#{ARCH}`)
+    - Kernel version: `#{`uname -r`.chomp}`
```
Comment (uberhacker) [inline bin/crew:1820]: Shouldn't need to nest the backticks.  This should work too?  `\`uname -r\`.chomp`

File: bin/crew [hunk 2]
```diff
@@ -431,6 +433,12 @@ def help(pkgName)
         crew search | grep -i audio".lightblue + " will display all packages with `audio` in the name or description.
         crew search git -v".lightblue + " will display packages with `git` in the name along with homepage, version and license.
     EOT
+  when "sysinfo"
+    puts <<~EOT
+      Show system infomation in Markdown style.
+
+      Usage: crew sysinfo
+    EOT
```
Comment (uberhacker) [inline bin/crew:441]: Add command to `README.md`?

File: bin/crew (hunk 1 above)
Comment (supechicken) [reply bin/crew:1820]: The outermost backticks are for markdown styling (`test`)

File: bin/crew (hunk 2 above)
Comment (supechicken) [reply bin/crew:441]: Done

**Comment by supechicken** [comment]
> Good idea! Maybe we should update the issue template and ask they run this command instead?

Added to #6563

**Pull request merged**